recipe/build.sh

```diff
@@ -42,6 +42,14 @@ ninja -C builddir/ -j ${CPU_COUNT}
 
 ninja -C builddir/ install
 
+# library_path in the ICD JSON points to the absolute path during the build
+# process. Patch it to point to the correct lib path.
+# Users should add $CONDA_PREFIX/share/vulkan/icd.d/lvp_icd.x86_64.json
+# to VK_DRIVER_FILES environment variable for the VK loader to pick up lavapipe.
+ICD_PATH=$(find "$PREFIX/share/vulkan/icd.d" -name 'lvp_icd.*.json' | head -n 1)
+jq '.ICD.library_path = "../../../lib/libvulkan_lvp.so"' $ICD_PATH > icd_temp.json
+mv icd_temp.json $ICD_PATH
```

Comment:
Did you try depending on:
https://github.com/conda-forge/khronos-opencl-icd-loader-feedstock

or maybe (this one is a stretch)
https://github.com/conda-forge/ocl-icd-system-feedstock

Reply:
These are OpenCL ICDs rather than the Vulkan ones. Unless I'm misunderstanding what your suggestion is 🤔

Reply:
Locally I can see that the file contains:

```
{
    "ICD": {
        "api_version": "1.4.305",
        "library_path": "/home/conda/feedstock_root/build_artifacts/debug_1742597260334/_h_env/lib/libvulkan_lvp.so"
    },
    "file_format_version": "1.0.0"
}
```

conda substitution will just look for the prefix, and it will be fine, so all this can be deleted.

Reply:
during a real build (non debug), conda will have

```
/prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_/lib....
```

and it will search for 
```
/prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_prefix_
```
and replace it with the true prefix.

Reply:
Oh, I didn't know that conda did the patching; that's interesting. Does it patch w/ an absolute path or local? Just concerned that if I have a portable app that contains a conda environment, that would be broken if it's moved if it's absolute.

Reply:
Making portable conda apps is a different problem. Definitely out of scope from this kind of « targeted fixes »

Reply:
That's fair. I'll just handle that on the app-side.